CMakeLists.txt

```diff
@@ -102,21 +102,18 @@ endif()
 if (VULKAN_HEADERS_INSTALL_DIR)
     list(APPEND CMAKE_PREFIX_PATH ${VULKAN_HEADERS_INSTALL_DIR})
 endif()
+if (VULKAN_VALIDATIONLAYERS_INSTALL_DIR)
+    list(APPEND CMAKE_INCLUDE_PATH ${VULKAN_VALIDATIONLAYERS_INSTALL_DIR}/include/vulkan)
+endif()
 
 find_package(VulkanHeaders REQUIRED CONFIG QUIET)
 
-set(VULKAN_VALIDATIONLAYERS_INSTALL_DIR "VALIDATIONLAYERS-NOTFOUND" CACHE PATH "Absolute path to Vulkan-ValidationLayers directory")
-
-# Cmake command line option overrides environment variable
-if(NOT VULKAN_VALIDATIONLAYERS_INSTALL_DIR)
-    set(VULKAN_VALIDATIONLAYERS_INSTALL_DIR $ENV{VULKAN_VALIDATIONLAYERS_INSTALL_DIR})
-endif()
-message(STATUS "Using Vulkan-ValidationLayers install located at ${VULKAN_VALIDATIONLAYERS_INSTALL_DIR}")
+find_file(VVL_INCLUDE_DIR vk_dispatch_table_helper.h REQUIRED CMAKE_FIND_ROOT_PATH_BOTH)
```

Comment:
Added CMAKE_FIND_ROOT_PATH_BOTH to fix android build. Since AndroidSDK tries to redirect all find_file commands by default to look into the Android SDK.

Reply:
This reflect what we currently do for packages as well. Just on a more granular level.